doc/building.md

```diff
@@ -2032,20 +2032,7 @@ First of all: Thank you! We gladly welcome your contribution.
 However, please bear in mind that the JDK is a massive project, and we must ask
 you to follow our rules and guidelines to be able to accept your contribution.
 
-The official place to start is the ['How to contribute' page](
-http://openjdk.org/contribute/). There is also an official (but somewhat
-outdated and skimpy on details) [Developer's Guide](
-http://openjdk.org/guide/).
-
-If this seems overwhelming to you, the Adoption Group is there to help you! A
-good place to start is their ['New Contributor' page](
-https://wiki.openjdk.org/display/Adoption/New+Contributor), or start
-reading the comprehensive [Getting Started Kit](
-https://adoptopenjdk.gitbooks.io/adoptopenjdk-getting-started-kit/en/). The
-Adoption Group will also happily answer any questions you have about
-contributing. Contact them by [mail](
-http://mail.openjdk.org/mailman/listinfo/adoption-discuss) or [IRC](
-http://openjdk.org/irc/).
+The official place to start is the [OpenJDK Developers’ Guide](https://openjdk.org/guide/).
```

Comment:
When changing this file, you also need to regenerate the `building.html`. See bottom of this file for instructions.

Reply:
See https://github.com/openjdk/jdk/pull/14716#discussion_r1288389758. I am supposed to check in just the modified HTML paragraph? All the modifications to `building.html`, even those unrelated to the Markdown change? All the modifications to other HTML files, even those unrelated to this patch?

Reply:
Different versions of pandoc will create different output. The version we currently use is defined in `make/devkit/createPandocBundle.sh`. If you are unable to use this version, I can generate the file for you.

Reply:
I did just that, here is my commit: https://github.com/erikj79/jdk/commits/JDK-8312882-contributing

Reply:
Ah, thanks. Yes I was using

```console
$ dpkg -l pandoc
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name           Version          Architecture Description
+++-==============-================-============-=================================
ii  pandoc         2.9.2.1-3ubuntu2 amd64        general markup converter
```

which I guess is too old. https://pandoc.org/installing.html#docker would be more predictable.